sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/SignInResult.java

```diff
@@ -5,6 +5,8 @@
 
 package com.microsoft.appcenter.identity;
 
+import com.microsoft.appcenter.utils.context.UserInformation;
```

Comment:
Optimize imports, it is suspicious that an import is added and not used.

Reply:
Will be fixed by rollbacking https://github.com/Microsoft/AppCenter-SDK-Android/pull/1047#discussion_r273196308